go.mod

```diff
@@ -15,13 +15,13 @@ require (
 	github.com/hashicorp/go-version v1.1.0
 	github.com/hashicorp/hcl v1.0.0 // indirect
 	github.com/hashicorp/terraform v0.12.6
-	github.com/pkg/errors v0.8.0 // indirect
 	github.com/satori/go.uuid v1.2.0
 	github.com/satori/uuid v0.0.0-20160927100844-b061729afc07
-	github.com/sirupsen/logrus v1.2.0 // indirect
 	github.com/terraform-providers/terraform-provider-azuread v0.6.0
 	github.com/tombuildsstuff/giovanni v0.3.2
 	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
 	golang.org/x/net v0.0.0-20190502183928-7f726cade0ab
 	gopkg.in/yaml.v2 v2.2.2
 )
+
+go 1.13
```

Comment:
Could we remove this as we are not on go 1.13 yet?

Reply:
Done.